src/etl/impl/LedgerLoader.hpp

```diff
@@ -256,6 +262,7 @@ class LedgerLoader {
                 backend_->writeAccountTransactions(std::move(insertTxResult.accountTxData));
                 backend_->writeNFTs(insertTxResult.nfTokensData);
                 backend_->writeNFTTransactions(insertTxResult.nfTokenTxData);
+                backend_->writeMPTHolders(std::move(insertTxResult.mptHoldersData));
```

Comment:
This move is not working as you expected it because your `writeMPTHolders` takes a const ref.